src/promptflow/promptflow/_utils/logger_utils.py

```diff
@@ -271,6 +278,18 @@ def _set_log_path(self):
                     handler = FileHandler(self.file_path)
                     log_handler.handler = handler
 
+    def _set_batch_run_flow_logs_handler(self):
```

Comment:
I think we need a description how we implement this feature, it is hard to understand the code here.

Reply:
Added. And don't create `FileHandlerConcurrentWrapper` for each __enter__ method